shipment.go

```diff
@@ -386,12 +408,14 @@ func (c *Client) GenerateShipmentFormWithOptionsWithContext(ctx context.Context,
 }
 
 // GetShipmentEstimatedDeliveryDate retrieves the estimated delivery date of each Rate via SmartRate.
-func (c *Client) GetShipmentEstimatedDeliveryDate(shipmentID, plannedShipDate string) (out []*EstimatedDeliveryDate, err error) {
+// Deprecated: Use EstimateDeliveryDateForShipment instead. This method will be removed in a future release.
```

Comment:
We should not make the breaking change here. This function has been here for a long time, and we should not break deprecate because the new naming is better and match to the new function name we are adding in SmartRate service